app/controllers/api_controller.rb

```diff
@@ -181,4 +181,20 @@ def update  # POST, PUT, PATCH
   def destroy # DELETE
     redirect_api_request(:destroy)
   end
+
+  def create_bundle_resource_blueprints(type, id, data)
+    blueprint = Blueprint.find(id)
+    service_catalog = parse_href(data["resources"].detect { |r| r.key?("service_catalog_href") }["service_catalog_href"])
+    service_dialog = parse_href(data["resources"].detect { |r| r.key?("service_dialog_href") }["service_dialog_href"])
+    service_templates = data["resources"]
+                        .select { |r| r.key?("service_template_href") }
+                        .collect { |st| parse_href(st["service_template_href"]) }
+    automate_entrypoints = data["resources"].detect { |r| r.key?("automate_entrypoints") }
+    blueprint.create_bundle(:service_templates => service_templates,
```

Comment:
the automate entry points are a hash (string => string). The keys are `Provision`, `Retirement`, and `Reconfigure`